pkg/store/filesystem/io.go

```diff
@@ -5,14 +5,15 @@ import (
 
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
+	merrors "github.com/micro/go-micro/v2/errors"
 )
 
 // Unmarshal file into record
 func (s Store) parseRecordFromFile(record proto.Message, filePath string) error {
 	file, err := os.Open(filePath)
 	if err != nil {
 		s.Logger.Err(err).Msgf("error reading file %v: file not found", filePath)
-		return err
+		return merrors.FromError(err)
```

Comment:
@refs Thinking about it, I don't like that we're logging here. Following the rule that errors should only be dealt with in one place, I'd vote for just returning the original error here without logging and then deal with logging and micro-errors in the places where the parsing and writing is used. Right now it's inconsistent and hard to follow, as we're e.g. in bundles.go sometimes returning an error an sometimes building a micro-error. Even if we have to write more code, I'd still like to remove building the errors in io.go and move it to bundles.go etc.
Let me know what you think.

Reply:
essentially I agree with this, I thought of this while building this. Otherwise errors will be logged twice; which is good for root cause analysis, but then we could include more context to the message.

Reply:
We should agree on a common practice. AFAICT it would make sense to wrap errors with the macro errors in the handlers.

Reply:
Let's go with wrapping an error's detail on handlers 👍 I'll update